docs/preparing_the_infrastructure.md

```diff
@@ -0,0 +1,84 @@
+
+  Follow the instructions in the [Getting Started Guide](https://access.redhat.com/documentation/en-us/red_hat_openshift_local/2.32/html/getting_started_guide/index) to install Red Hat OpenShift Local.
+
+> [!IMPORTANT]
+> 1. OpenShift local comes with three presets that represent managed container runtimes: `openshift`, `microshift`, and `podman`. We will use the `openshift` preset to create the local OpenShift cluster.
+> 1. Make sure you start the OpenShift Local cluster with at least 18GB of RAM and 6 CPU cores via `crc start --memory 18432 --cpus 6`
+
+Once the OpenShift Local cluster is up and running, you can use it as the edge cluster in the proof of concept and continue with the installation of the components in the next sections.
+
+## Installation of Red Hat OpenShift Pipelines
+
+Building an inference service container with a trained model can be done using Red Hat OpenShift Pipelines. Follow the instructions in [pipelines/README.md](../pipelines/README.md) to install Red Hat OpenShift Pipelines and configure the core cluster to use it to build the inference service containers.
+
+## Installation of Red Hat Advanced Cluster Management for Kubernetes (ACM)
+
+> [!IMPORTANT]
+> The following setup steps need to be done as an admin user with `cluster-admin` privileges.
+We will describe how to do them using the OpenShift Console but there may be other ways to achieve the same results.
+
+1. On the ACM hub cluster, install the Advanced Cluster Management for Kubernetes operator.
+   In the OpenShift Console of the cluster, find and install the operator
+   in Menu > Operators > OperatorHub.
+1. Create the MultiClusterHub.
+   In the OpenShift Console in Menu > Operators > Installed Operators > Advanced Cluster Management for Kubernetes >
+   the MultiClusterHub tab, use the "Create MultiClusterHub" button to create its configuration.
+1. Refresh the OpenShift Console page.
+1. Create a cluster set for your near edge clusters.
+   In the OpenShift Console this can be done in All Clusters menu > Infrastructure > Clusters > Cluster sets tab.
+   This repository assumes the name `poc-near-edge` in `clusterSets`
+   in [`acm/registration/near-edge/base/near-edge.yaml`](../acm/registration/near-edge/base/near-edge.yaml)
+   so update that YAML file if you use a different name.
+1. Import the near edge cluster(s).
+   In the OpenShift Console this can be done in All Clusters menu > Infrastructure > Clusters > Cluster list tab.
+   In the Cluster set popup menu, select the cluster set you created for the near edge clusters.
+   If you use the Import mode of "Enter your server URL and API token for the existing cluster",
+   obtain the address of the server and the token on the other cluster's OpenShift Console
+   at the top-right corner user menu > Copy login command, for an admin user.
+
+> [!IMPORTANT]
+> To import the OpenShift Local clusters you must use the "Run import commands manually" import mode.
```

Comment:
I guess this is true for all clusters that the hub can't connect directly to, right?

Reply:
I don't think so, I think the other mode works if the edge-cluster is discoverable IIUC.

Reply:
We're saying the same thing from different angles :upside_down_face: